src/overcooked_demo/server/game.py

```diff
@@ -37,18 +37,18 @@ def fix_bc_path(path):
     For now the solution is to include the saved BC model and fix the relative path to the model in the config.pkl file
     """
 
-    import pickle
+    import dill
```

Comment:
Why did you switch to dill? Just curious!

Reply:
Okay so this is a long story.... I was running into a weird bug that no agents would load when running the web demo from the newly reproduced agents, and I tried a lot of things, including changing pickle to dill for the reason [documented here](https://github.com/HumanCompatibleAI/overcooked_ai/blob/15e50b7228f2b58220636e411905a668d79d12d5/src/human_aware_rl/rllib/rllib.py#L822). 
It turns out that the actual reason was network delay, by which I mean since I was downloading a lot of checkpoints files from the server at once, and they didn't finish downloading until a couple of minutes later, but the directories and config.pkl files became visible before the downloads were complete, meaning the reason why the load failed was because the files were missing contents......
Now everything works and I just didn't bother changing it back to pickle since I found this comment and it seemed reasonable.

Reply:
Damn that sounds very annoying! 😅